contrib/spec/openarc.spec.in

```diff
@@ -31,111 +56,145 @@ using libopenarc.
 %package -n libopenarc-devel
 Summary: Development files for libopenarc
 Group: Development/Libraries
-Requires: libopenarc
+Requires: lib%{name}%{?_isa} = %{version}-%{release}
 
 %description -n libopenarc-devel
 This package contains the static libraries, headers, and other support files
 required for developing applications against libopenarc.
 
 %prep
-%setup
+%autosetup -p1
 
 %build
-# Required for proper OpenSSL support on some versions of RedHat
-if [ -d /usr/include/kerberos ]; then
-	INCLUDES="$INCLUDES -I/usr/include/kerberos"
-fi
-./configure --prefix=%{_prefix} --sysconfdir=%{_sysconfdir} --libdir=%{_libdir} --mandir=%{_mandir} @SPECCONFIGURE@ CPPFLAGS="$INCLUDES"
-
-make
+autoreconf --install
```

Comment:
once we have released tarballs that include configure, you are right.  As it stands, ./configure is specifically excluded by .gitignore, so we must create it.